### DISCUSSION TITLE
Chess TUI official team on lichess

### DISCUSSION THREAD

**Discussion by shuvashish76** [thomas-mauran/chess-tui#112] [Ideas]
Create a lichess [team](https://lichess.org/team/all) so that when everything gets ready for lichess we can get feedback from users on our lichess team forum page.
I think it's the best way to get feedbacks from lichess users. Also not all lichess players like to create account on GitHub for their reports.
I understand it's not ready _(only planned)_ but still I think no harm in building little lichess community there.
Maybe some team specific arenas in future to popularize this project.

**Comment by thomas-mauran** [comment]
Wow another great idea mate ! Didn't know this was a thing I will definitely create it

**Comment by thomas-mauran** [comment]
Hey @shuvashish76 https://lichess.org/team/chess-tui

Here is the team if you wanna join it
